SwanPortAllocator/swanportallocator/portallocator.py

```diff
@@ -62,35 +63,46 @@ def __init__(self, log):
     def get_ports(self, process, n):
         """
             Return 'n' available ports and assign them to 'process' pid.
-            If the 'process' already asked for ports, and the information was not recycled, return the same ports.
             Raises NoPortsException if all ports are in use.
         """
 
         if process in self.clients:
```

Comment:
Single line if?

Reply:
You mean `stored_ports = self.clients[process]['ports'] if process in self.clients else []`

Reply:
Done.